cdci_data_analysis/analysis/instrument.py

```diff
@@ -165,9 +145,8 @@ def test_has_input_products(self, config,instrument,logger=None):
         if self.data_server_query_class is not None:
             return self.data_server_query_class(config=config,instrument=self).test_has_input_products(instrument,logger=logger)
 
-
-
-    def run_query(self,product_type,
+    def run_query(self, product_type,
+                  query_name,
```

Comment:
why did this change? I sorry, I dont follow

Reply:
I moved it here since I was in need to extract the name of the query from the product type, and in order not to repeat the same operation twice I decide to pass it as an argument. I will probably roll back, since I realized that there is not such a big advantage with it.

Reply:
#33 opened for the purpose